ui/src/app/components/apps-panel/expanded-app.spec.tsx

```diff
@@ -38,6 +48,13 @@ const workspace = {
 const onClickRuntimeConf = jest.fn();
 const onClickDeleteRuntime = jest.fn();
 
+function minus<T>(a1: T[], a2: T[]): T[] {
+  return a1.filter((e) => !a2.includes(e));
+}
+const ALL_STATUSES = Object.keys(AppStatus)
```

Comment:
as we've discussed, this can be problematic ... but it looks OK in this case!

Reply:
I'll avoid relying on this in non-test code 👍